src/test/components/__snapshots__/ButtonWithPanel.test.js.snap

```diff
@@ -44,24 +44,6 @@ exports[`shared/ButtonWithPanel opens the panel when the button is clicked and c
   >
     My Button
   </button>
-  <div
-    class="arrowPanelAnchor"
-  >
-    <div
-      class="arrowPanel panel"
-    >
-      <div
-        class="arrowPanelArrow"
-      />
-      <div
-        class="arrowPanelContent"
-      >
-        <div>
-          Panel content
-        </div>
-      </div>
-    </div>
-  </div>
```

Comment:
These snapshots were incorrect before, because we weren't waiting until the panel was closed.